### PR TITLE
Transform React.propTypes -> PropTypes

### PR DESCRIPTION
**Pull request by dekdekbaloo**

### CODE REVIEW

**Review by dtinth — APPROVED** [review]

**Comment by dekdekbaloo** [comment]
@dtinth I can't merge this myself since I don't have write access TT 555

**Comment by dtinth** [comment]
Thanks a lot!

I gave you write access.

After merging, to release a new version, run `git checkout master`, `git pull`, `npm version 0.15485863.0`, `git push --follow-tags`. This will release a new version to npm automatically.

**Pull request merged**
